File: CS/ReadOnlyTabStopRemover.Module/Updater.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Updating;

namespace ReadOnlyTabStopRemover.Module {
    public class Updater : ModuleUpdater {
        public Updater(IObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            DemoObject obj1 = ObjectSpace.CreateObject<DemoObject>();
            obj1.Name = "DemoObject1";
            obj1.DisableSimpleProperty = true;
            obj1.Save();
            DemoObject obj2 = ObjectSpace.CreateObject<DemoObject>();
            obj2.Name = "DemoObject2";
            obj2.DisableSimpleProperty = false;
            obj2.Save();
        }
    }
}

File: VB/ReadOnlyTabStopRemover.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

Namespace ReadOnlyTabStopRemover.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As ObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim obj1 As DemoObject = ObjectSpace.CreateObject(Of DemoObject)()
			obj1.Name = "DemoObject1"
			obj1.DisableSimpleProperty = True
			obj1.Save()
			Dim obj2 As DemoObject = ObjectSpace.CreateObject(Of DemoObject)()
			obj2.Name = "DemoObject2"
			obj2.DisableSimpleProperty = False
			obj2.Save()
		End Sub
	End Class
End Namespace